MODULE.bazel

```diff
@@ -100,6 +100,14 @@ bazel_dep(name = "spdlog", version = "1.15.1")
 bazel_dep(name = "tcmalloc", version = "0.0.0-20250331-43fcf6e")
 bazel_dep(name = "zlib", version = "1.3.1.bcr.5")
 
+
+bazel_dep(name = "qt-bazel")
+git_override(
+    module_name = "qt-bazel",
+    commit = "ab451ca5fca51433c48168f4c404a43e15b97f54",
+    remote = "https://github.com/The-OpenROAD-Project/qt_bazel_prebuilts",
```

Comment:
Please add some description about the magic that makes this all work.

Reply:
Done.